demisto_sdk/tests/test_files/invalid_py3_XSOARLinter_no_test_module.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict, Tuple, Callable
```

Comment:
No need to add another test file, just remove the test-module check in the warning file: `XSOAR_LINTER_PY3_INVALID_WARNINGS` and add the message to the results list.
Currently there is no section of warning messages in the `base` level , so just add one.